content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -116,12 +116,12 @@ be down for the same period as the previous example (`08:29:00` to `10:21:00`,)
 The CronJob is only responsible for creating Jobs that match its schedule, and
 the Job in turn is responsible for the management of the Pods it represents.
 
-## New controller
+## Controller version {#new-controller}
 
-There's an alternative implementation of the CronJob controller, available as an alpha feature since Kubernetes 1.20. To select version 2 of the CronJob controller, pass the following [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) flag to the {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}.
+Starting from Kubernetes 1.21 the version 2 of the CronJob controller is now the default implementation. To disable the new controller and use the old one pass the following [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) flag to the {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}.
```

Comment:
Starting with Kubernetes v1.21, the second version of the CronJob controller is the default implementation. To disable the second Cronjob controller and use the first CronJob controller, pass the `CronJobControllerV2` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) flag to the {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}, and set this flag to `false`. For example:

Reply:
My tweaks to that:
```suggestion
Starting with Kubernetes v1.21, the second version of the CronJob controller is the default implementation. To disable the default CronJob controller and use the original CronJob controller instead, pass the `CronJobControllerV2` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) flag to the {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}, and set this flag to `false`. For example:
```

BTW: wrapping helps with reviews (but the original is also not wrapped).

Reply:
> BTW: wrapping helps with reviews (but the original is also not wrapped).

Yeah, I don't like when docs are written like that :disappointed: Fix it now with update.